airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -1498,6 +1498,7 @@ def insert_job(
         job_id: Optional[str] = None,
         project_id: Optional[str] = None,
         location: Optional[str] = None,
+        is_async: bool = False,
```

Comment:
i think we should rename `is_async`... it doesn't seem quite right because you are not _describing_ something (i.e. whether it _is_ async or not), but rather _instructing_ the func whether to make the call in an async way or not (so maybe `use_async` would be better).

but there's a suggestion from @uranusjr to use `nowait` which i think is probably better because it avoids the word `async` which is a bit fraught.

Reply:
agreed , made changes in the code accordingly